AirCasting/CreateSessionViews/SelectDeviceView.swift

```diff
@@ -61,6 +61,9 @@ struct SelectDeviceView: View {
             selected = 0
             #warning("Handle that mobileWasTapped is somehow public")
             emptyDashboardButtonTapped.mobileWasTapped = false
+            isBTButtonActive(completion: { value in
```

Comment:
How about `isBTButtonActive({ isBTButtonAvailable = $0 })? However, I don't like this buttonActive part of the name and buttonAvailable either. Maybe the name of the fuction could be isBTConnectionAvailable and variable can be called isBTButtonEnabled (or disabled)?

Reply:
I agree, `isConnectionAvailable` is better, currently I can't really tell what's going on here, why is button active or inactive, changing func name to `isAirBeamConnectionAvailable` or something will make it super clear